src/backends/kubernetes/kubernetes_backend.py

```diff
@@ -78,7 +78,7 @@ def get_file(self, tank_index: int, service: ServiceType, file_path: str):
         Read a file from inside a container
         """
         pod_name = self.get_pod_name(tank_index, service)
-        exec_command = ["cat", file_path]
+        exec_command = ['sh', '-c', f'cat "{file_path}" | base64']
```

Comment:
@m3dwards what's you'r professional opinion on bas64-ing files out of k8s? Kinda nasty, but we are forced into returning `str`, so I don't really see a better way...

Reply:
When I last looked into it, I think I read that some people were TARing but I don't know how that helps with the bytes problem as it would just be bytes of a tar?

If base64 works then we can keep it like that.